server/settings_test.go

```diff
@@ -192,6 +195,58 @@ func TestSettings(t *testing.T) {
 				assert.True(t, resg.Payload.Settings.SttEnabled)
 			})
 
+			// Verify Failed checks alerts in Alertmanager
+			t.Run("VerifyFailedChecksInAlertmanager", func(t *testing.T) {
+				defer restoreDefaults(t)
+				// Enabling STT
+				res, err := serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
+					Body: server.ChangeSettingsBody{
+						EnableStt: true,
+					},
+					Context: pmmapitests.Context,
+				})
+				require.NoError(t, err)
+				assert.True(t, res.Payload.Settings.TelemetryEnabled)
+
+				activeAlerts := true
+				silencedAlerts := false
+				alertsCount := 0
+
+				// 120 sec ping for failed checks alerts to appear in alertmanager
+				for i := 0; i < 120; i++ {
+					res, err := amclient.Default.Alert.GetAlerts(&alert.GetAlertsParams{
+						Active:   &activeAlerts,
+						Silenced: &silencedAlerts,
+						Context:  pmmapitests.Context,
+					})
+					require.NoError(t, err)
+					if len(res.Payload) != 0 {
+						for _, v := range res.Payload {
+							// Verify that there is description in response
+							if _, ok := v.Annotations["description"]; !ok {
+								assert.True(t, false, "Description not met in response")
```

Comment:
I think you want `assert.True(ok, …` without `if`.

Reply:
done , thanks